pkg/cmd/openshift.go

```diff
@@ -30,6 +34,7 @@ func openshiftCmd(vals *types.CmdValues) *cobra.Command {
 				fmt.Fprintln(os.Stderr, err.Error())
 				os.Exit(1)
```

Comment:
You you are right, with help it is hard to see error message :(
Keep it that way.
Can you maybe just add comment that this is reason for os.Exit there?